pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -70,7 +74,7 @@ func (pl *VolumeZone) Name() string {
 // the node to have more zone-label constraints (for example, a hypothetical replicated
 // volume might allow region-wide access)
 //
-// Currently this is only supported with PersistentVolumeClaims, and looks to the labels
+// Currently this is only supported with PersistentVolumeClaims and generic ephemeral volumes, and looks to the labels
```

Comment:
This predicate is deprecated in favor of volume topology now. We don't have to extend support for generic ephemeral volumes here.

Reply:
So shall I remove the entire commit which touches `volume_zone.go`?

Reply:
I've removed it and can bring it back if desired.